src/org/opendatakit/briefcase/ui/BriefcaseCLI.java

```diff
@@ -248,7 +251,8 @@ public void run() {
                         break;
                     } catch (IOException e) {
                         String msg = "The supplied PEM file could not be parsed.";
-                        log.error(msg, e);
+                        System.err.println("msg");
```

Comment:
This will print _msg_ literally instead of the `msg` variable

Reply:
And I think `msg` can be inlined.

Reply:
Thank you! This is why code review is so important.